.claude/skills/debug-viz/scripts/debug_viz.py

```
@@ -0,0 +1,211 @@
+    parser.add_argument("--open", action="store_true", help="Open in browser")
+
+    args = parser.parse_args()
+
+    html_path, debug_info = generate_debug_html(
+        args.module,
+        args.variable,
+        args.depth,
+        args.separate_outputs,
+    )
+
+    print(f"\n=== HTML Generated ===")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)